notebooks/demo_latticemaze.ipynb

```diff
@@ -211,17 +225,20 @@
             "outputs": [],
             "source": [
                 "MazePlot(maze).plot()\n",
-                "plt.show()"
+                "plt.show()\n",
```

Comment:
Should we just get rid of the ```MazePlot.show()``` function?

If I understood correctly, we go with
```
Mazeplot.plot()
plt.show()
```

instead of 

```MazePlot.show()``` for better testing within notebooks.

Reply:
I will go ahead and remove `MazePlot.show()` -- if we consume it somewhere in the experiments repo notebooks, the notebook tests should alert us to any issues